### ISSUE TITLE
a question about training

### ISSUE THREAD

**Issue by limaodaxia** [marco-rudolph/AST#21]
Hi, @marco-rudolph, how is going recently:)

Should I set the same batchsize and learning rate for all category in the dataset?
In academic paper, Is it necessary that setting the same learning rate  to demonstrate the effectiveness of the model?

Limao
Best wishes!

**Comment by marco-rudolph** [comment]
Hi,

thanks, I'm good.

As you could normally not make a hyperparameter tuning in reality (without having anomalies), it has become a silent convention to use the same hyperparameter for every class and tune the (same) hyperparameter over all classes. The intuition is that hyperparameters which work well for n classes, should work well for the n+1th class.

Best,
Marco

**Comment by limaodaxia** [comment]
Thank you very much, I got it.
